pkgs/by-name/be/beeref/package.nix

```diff
@@ -0,0 +1,56 @@
+{
+  lib,
+  python3Packages,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "beeref";
+  version = "0.3.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "rbreu";
+    repo = "beeref";
+    tag = "v${version}";
+    hash = "sha256-GtxiJKj3tlzI1kVXzJg0LNAUcodXSna17ZvAtsAEH4M=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+    wheel
+  ];
+
+  dependencies = with python3Packages; [
+    exif
+    lxml
+    rectangle-packer
+    pyqt6
+  ];
+
+  pythonRelaxDeps = [
+    "pyqt6"
+    "rectangle-packer"
+    "lxml"
+  ];
+
+  pythonRemoveDeps = [ "pyqt6-qt6" ];
+
+  pythonImportsCheck = [ "beeref" ];
```

Comment:
Can you try adding `versionCheckHook` to `nativeCheckInputs`?

Reply:
Isn't it supposed to be in `nativeInstallCheckInputs`? Though apparently it is not working in it.

Reply:
Got it, thanks for the explanation!